packages/fiori/src/themes/UploadCollection.css

```diff
@@ -26,19 +17,15 @@
 	background-color: var(--sapGroup_ContentBackground);
 }
 
-.uc-no-files .icon-container {
-	height: 8rem;
-}
-
 .uc-no-files .icon-container [ui5-icon] {
-	font-size: 6rem;
-	width: 6rem;
-	height: 6rem;
+	font-size: 8rem;
+	width: 8rem;
```

Comment:
I think that the correct width & height for the icon is 6rem.

Moreover, the space over the ```No files found``` text and underneath should be the same.

Reply:
Done. Styles are updated according to the internal spec https://wiki.wdf.sap.corp/wiki/pages/viewpage.action?pageId=2036062597